test/qa/test_settings.py

```diff
@@ -119,3 +120,251 @@ def test_technology_set_options(tech, proto, obfuscated):
         assert ovpn_list
     else:
         assert not ovpn_list
+
+
+@pytest.mark.parametrize(("tech", "proto", "obfuscated"), lib.TECHNOLOGIES)
+def test_set_defaults_when_logged_in_1st_set(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
+    daemon.restart() # Temporary solution to avoid Firewall staying enabled in settings - LVPN-4121
+
+    sh.nordvpn.set.firewall("off")
+    sh.nordvpn.set.routing("off")
+    sh.nordvpn.set.dns("1.1.1.1")
+    sh.nordvpn.set.analytics("off")
+    sh.nordvpn.set.ipv6("on")
+    sh.nordvpn.set.notify("on")
+
+    assert not settings.get_is_firewall_enabled()
+    assert not settings.get_is_routing_enabled()
+    assert not settings.get_is_dns_disabled()
+    assert not settings.get_are_analytics_enabled()
+    assert settings.get_is_ipv6_enabled()
+    assert settings.get_is_notify_enabled()
+
+    if obfuscated == "on":
+        assert settings.get_is_obfuscated()
+    else:
+        assert not settings.get_is_obfuscated()
+
+    assert MSG_SET_DEFAULTS in sh.nordvpn.set.defaults()
+
+    assert settings.get_default_application_settings()
+
+
+@pytest.mark.parametrize(("tech", "proto", "obfuscated"), lib.STANDARD_TECHNOLOGIES) # Only using standard technologies here because of "LVPN-4601 - Enabling Auto-connect disables Obfuscation"
+def test_set_defaults_when_logged_in_2nd_set(tech, proto, obfuscated):
```

Comment:
is `2nd_set` related to `1st_set`? are they expected to run in sequence?

Reply:
I made 2 sets since some settings cannot be disabled/enabled together e.g. routing cannot be disabled when meshnet is enabled, etc., as well as because of that bug we have where enabling auto connect disables obfuscation. So 2 sets were made in order to check if all settings, after they are changed, can be reset to default value.